docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -493,20 +483,16 @@ La configuración anterior genera la siguiente barra lateral:
 <SidebarPreview
 	config={[
 		{
-			label: 'Guías',
+			label: 'Constelaciones',
 			collapsed: true,
 			items: [
-				{ label: 'Componentes', link: '/guides/components/' },
-				{ label: 'Internacionalización (i18n)', link: '/guides/i18n/' },
+
+				{ label: 'Carina', link: '' },
+				{ label: 'Centaurus', link: '' },
 				{
-					label: 'advanced',
+					label: 'estacional',
```

Comment:
```suggestion
					label: 'seasonal',
```

In the case of a nested folder in an autogenerated group, the folder name will be used as we do not have an API yet for this and so far, the directory name was not translated in this page.